.vscode/twenty.code-workspace

```diff
@@ -42,10 +42,59 @@
     },
   ],
   "settings": {
```

Comment:
Is it intended?

Reply:
Yes, actually when opening VSCode in workspace mode, lot of thing are not working properly and jest test are auto-run on each save 🤯